src/Symfony/Component/HttpClient/ClientState/CurlClientState.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpClient\ClientState;
```

Comment:
what about naming this namespace `Internal` like done in `VarExporter`?

Reply:
Will do.